src/Host/Client/Impl/BrokerServices/WebService.cs

```diff
@@ -81,17 +81,18 @@ private Task RepeatUntilAuthenticatedAsync(Func<CancellationToken, Task> action,
             }
         }
 
-        public Task<TResponse> HttpGetAsync<TResponse>(UriTemplate uriTemplate, CancellationToken cancellationToken = default(CancellationToken), params object[] args) =>
-           HttpGetAsync<TResponse>(MakeUri(uriTemplate, args), cancellationToken);
+        //public Task<TResponse> HttpGetAsync<TResponse>(UriTemplate uriTemplate, CancellationToken cancellationToken = default(CancellationToken), params object[] args) =>
```

Comment:
nope

---
In reply to: [111815999](https://github.com/Microsoft/RTVS/pull/3419#discussion_r111815999) [](ancestors = 111815999)